tests/mocks/examples_ttl.py

```diff
@@ -632,15 +675,21 @@
 @prefix dct: <http://purl.org/dc/terms/> .
 @prefix ns1: <https://data.norge.no/vocabulary/modelldcatno#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
 
 <http://uri.com> a ns1:InformationModel ;
+    dct:hasFormat <http://publications.europa.eu/resource/authority/file-type/JSON> ;
```

Comment:
Dette er vel ikkje korrekt? 
dct:hasFormat skal peike på eit foaf:Document, ikkje eit begrep.

Reply:
Den peker på en foaf-document-ressurs som er lengre nede i rdf'en, men jeg satte identifieren til foaf-documentet som den uri'en men det kan jeg endre

Reply:
Hei, her har jeg gjort en feil. Rett det som Stig sier, det er dct:format, som er en egenskap til foaf:Document som forholder seg til EU-vokabularet.

Reply:
Blander dere ikke nå _dct:hasFormat_ på modelldcatno:InformationModel (med ragne foaf:Document) med _dct:format_ på foaf:Document? Jeg endrer som sagt identifieren slik at det ikke blir forvirrende..

Reply:
Jo, hasFormat ligger på InformationModel. Her er et eksempel, sikkert bedre å ha brukt blank node for å framstille dette:
![image](https://user-images.githubusercontent.com/17720749/117118669-59007f00-ad91-11eb-9338-9f3846a9cd7c.png)

Reply:
Hei beklager, det er jeg som blander her. :blush: Jeg skal rette dette slik som dere sier. :-)